packages/config/config/devices/0x031e/lzw30.json

```diff
@@ -238,16 +243,50 @@
 				}
 			]
 		},
-		"8": {
-			"label": "LED Strip Effect",
-			"description": "Calculated sum of parameters as per overview section",
-			"valueSize": 4,
+		"13": {
+			"$if": "firmwareVersion >= 1.17",
+			"label": "Special Load Type",
+			"description": "Can be used in certain 3-way dumb switch setups where the load is confusing the switch as to which state it should be in.",
+			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 83823359,
+			"maxValue": 1,
 			"defaultValue": 0,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Detect Load Type",
+					"value": 0
+				},
+				{
+					"label": "Manually set for special load type",
+					"value": 1
+				}
+			]
+		},
+		"51": {
+			"$if": "firmwareVersion >= 1.19",
+			"label": "Instant On",
+			"description": "Enables instant on (ie: disables the 700ms button delay).",
```

Comment:
The description does not seem to add value in addition to the label and the options. Please remove.

Reply:
Removed